plugins/isp/internode_usage

```diff
@@ -38,7 +38,7 @@ multiple times, by symlinking it as 'internode_usage_SERVICEID'.
 =head1 CACHING
 
 As the API is sometimes flakey, the initial service information is cached
-locally, with a one-hour lifetime, before hitting the base API again. However,
+locally, with a day's lifetime, before hitting the base API again. However,
```

Comment:
I assume, that this (unrelated) change somehow sneaked into your proposal accidentally.

I am fine with merging it here anyway, if you are confident, that it is indeed an improvement.

Reply:
It's somewhat related, as I used the caching code as a base, and realised the doc was incorrect, so I fixed it at the same time (: